src/main/java/cow/farm/utils/QuantityCheckingList.java

```diff
@@ -0,0 +1,16 @@
+package cow.farm.utils;
+
+import cow.farm.utils.LukaszGenericArrayList;
+
+import java.util.HashMap;
+
+public class QuantityCheckingList<T> extends LukaszGenericArrayList<T> {
+
+	public QuantityCheckingList() {
+	}
+
+	public HashMap<T , Integer> getQuantityOfEachObject(){
```

Comment:
Zadziała po merge...ale faktycznie, skopiowałem plik, zrobiłem test, a potem cofnąłem zmiany, bo przecież tamten plik był mergowany innym commitem.
Teraz widzę, ze dla sprawdzającego to bez sensu. Mogłem zrobić merge #10 i potem commit. poprawiam w drodze do domu.